app/controllers/application_controller.rb

```diff
@@ -28,12 +28,20 @@ def after_sign_in_path_for(user)
       edit_profile_path
     elsif request.referrer.present? && request.referrer != new_user_session_url
       request.referrer
-    elsif user.admin?
-      admin_events_path
-    elsif user.proposals.any? #speaker
```

Comment:
I don't think we want to get rid of the rule here for speakers with proposals; they should land on the My Proposals page. It's only a problem when a staff member also has a submitted proposal.

Basically, you just want to check for whether someone is staff *before* you check for whether they have any submitted proposals. So in the case where both things are true, one takes priority over the other.